beetsplug/subsonicplaylist.py

```diff
@@ -0,0 +1,138 @@
+# -*- coding: utf-8 -*-
+# This file is part of beets.
+# Copyright 2019, Joris Jensen
+#
+# Permission is hereby granted, free of charge, to any person obtaining
+# a copy of this software and associated documentation files (the
+# "Software"), to deal in the Software without restriction, including
+# without limitation the rights to use, copy, modify, merge, publish,
+# distribute, sublicense, and/or sell copies of the Software, and to
+# permit persons to whom the Software is furnished to do so, subject to
+# the following conditions:
+#
+# The above copyright notice and this permission notice shall be
+# included in all copies or substantial portions of the Software.
+
+from __future__ import absolute_import, division, print_function
+
+from hashlib import md5
+import xml.etree.ElementTree as ET
+import random
+import string
+from urllib.parse import urlencode
+
+import requests
+from beets.dbcore.query import SubstringQuery
+
+from beets.dbcore import AndQuery
+
+from beets.ui import Subcommand
+
+from beets.plugins import BeetsPlugin
```

Comment:
Any particular reason for the double-spacing around these imports?

Reply:
Guess it was from Pycharm autoformat